ebl/v3/EBL_v3.0.0.yaml

```diff
@@ -5355,14 +5350,13 @@ components:
           type: string
           pattern: ^\S(?:.*\S)?$
           maxLength: 35
-          # Extended description of carrierBookingReference compared to DCSA_DOMAIN description
           description: |
             The associated booking number provided by the carrier for this `Consignment Item`.
           example: ABC709951
         descriptionOfGoods:
           type: array
           description: |
-            An array of lines accurately and properly describing the cargo being shipped in the container(s) as provided by the shipper.
+            A plain language description that is precise enough for Customs services to be able to identify the goods. General terms (i.e. 'consolidated', 'general cargo' 'parts' or 'freight of all kinds') or not sufficiently precise description cannot be accepted.
```

Comment:
In the jira issue I see this part also in the description of DangerousGoods - Where the declarant provides the CUS code for chemical substances and preparations, Member States may waive the requirement of providing a precise description of the goods.

Reply:
`descriptionOfGoods` does not exist on DG. Not sure what you mean?